docs/modules/ROOT/pages/explanations/release.adoc

```diff
@@ -1,4 +1,6 @@
-= Releasing k8up
+= Releasing K8up
+
+IMPORTANT: Before pushing a new Git tag, please adjust the version number in https://github.com/vshn/k8up/blob/master/docs/antora.yml[antora.yml]. This isn't automated yet.
```

Comment:
Since we never know whether it's ever going to be automated, I would remove this sentence.
```suggestion
IMPORTANT: Before pushing a new Git tag, please adjust the version number in https://github.com/vshn/k8up/blob/master/docs/antora.yml[antora.yml].
```